app/AccountCreation.tsx

```diff
@@ -0,0 +1,106 @@
+import { ReactElement, useState } from "react";
+import { Button, Form } from "react-bootstrap";
+import { userNameAlreadyExists } from "@/server/getCredentials";
+import styled from "styled-components";
+import { PageHeader } from "./ui/PageHeader";
+import { FormGroup } from "./ui/FormGroup";
+import { FormControl } from "./ui/FormControl";
+
+export function AccountCreation(props: {
+  backToLogin: () => void;
```

Comment:
this component shouldn't know about the existence of a login page

Reply:
https://github.com/auberginearms/starterApp/pull/4/commits/fcd64eab0da9ef077a88d56b9ff57e4342b7add5